spring-cloud-deployer-cloudfoundry/src/test/java/org/springframework/cloud/deployer/spi/cloudfoundry/CloudFoundryTaskLauncherTests.java

```diff
@@ -517,6 +537,11 @@ public void testCommand() {
 		assertThat(command).isEqualTo("command-val test-command-arg-1 a=b run.id=1 run.id\\\\\\(long\\\\\\)=1 run.id\\\\\\(long=1 run.id\\\\\\)=1");
 	}
 
+	@Test
+	void getLog() {
```

Comment:
We are missing test coverage for these cases:
* null entries in the log
* ensuring we are getting the task guid from the proper place (correct guid)
* negative cases where the task guid was not found/retrieved

Reply:
Also, no coverage added for CF app deployer changes